DropInDemo/Demo Base/Settings/Settings.bundle/Root.plist

```diff
@@ -110,6 +110,34 @@
 				<string>Optional</string>
 			</array>
 		</dict>
+        <dict>
+            <key>Type</key>
+            <string>PSMultiValueSpecifier</string>
+            <key>Title</key>
+            <string>3DS Requested Version</string>
+            <key>Key</key>
+            <string>BraintreeDemoSettingsThreeDSecureVersionDefaultsKey</string>
+            <key>Values</key>
+            <array>
+                <integer>0</integer>
+                <integer>1</integer>
+                <integer>2</integer>
+            </array>
+            <key>DefaultValue</key>
+            <integer>0</integer>
+            <key>Titles</key>
+            <array>
+                <string>Off</string>
```

Comment:
I know that `Off` was part of the segmented control UI, but it feels like it's in the wrong place here. The `Off` scenario should probably be handled by the `BraintreeDemoTransactionServiceThreeDSecureRequiredStatus` setting.